bin/generate

```diff
@@ -147,7 +147,7 @@ function generate_hub_commit() {
 
   HUB_DIR="looker-hub"
   GENERATION_DISALLOW_LIST="/app/lookml-generator/GENERATION_DISALLOW_LIST"
-  NAMESPACE_ALLOWLIST="/app/lookml-generator/namespace_allowlist.yaml"
+  NAMESPACE_ALLOWLIST="/app/lookml-generator/namespaces-allowlist.yaml"
```

Comment:
I didn't see the PR go by where this changed? I'm sure it's fine.

Reply:
That was here: https://github.com/mozilla/lookml-generator/pull/93